fmutils.go

```diff
@@ -21,6 +21,11 @@ func Prune(msg proto.Message, paths []string) {
 	NestedMaskFromPaths(paths).Prune(msg)
 }
 
+// Override - overrides all of the fields listed in paths in dest msg using the values from src msg.
+func Override(src, dest proto.Message, paths []string) {
```

Comment:
How about renaming this function to `Overwrite`? What do you think?

Reply:
yep, sg to me.